dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -15,13 +15,20 @@ def index
 
     summarized_plc_enrollments = Plc::UserCourseEnrollment.where(user: current_user).map(&:summarize)
 
+    if courses_completed.include?(Pd::Workshop::COURSE_CSF)
+      enrollment = Pd::Enrollment.where(pd_workshop_id: ended_workshops.where(course: Pd::Workshop::COURSE_CSF).map(&:id), email: current_user.email).last
```

Comment:
These are not ordered, so last won't necessarily be consistent.

Please add a test to verify this is working correctly, that it gets the latest enrollment for the given user in a CSF ended workshop, disregarding enrollments from other users and other workshops.

Make sure you also check `user_id: current_user.id`. In case a user changes email after the fact: `where('email = ? OR user_id = ?', current_user.email, current_user.id)`

This logic might better fit the enrollment model, with testing there. At least the user part.